repos/system_upgrade/el8toel9/actors/checkarmbootloader/libraries/checkarmbootloader.py

```diff
@@ -0,0 +1,47 @@
+from leapp import reporting
+from leapp.libraries.common.config.architecture import ARCH_ARM64, matches_architecture
+from leapp.libraries.common.config.version import get_source_version, get_target_version
+from leapp.libraries.stdlib import api
+
+
+def _inhibit_upgrade():
+    title = 'Upgrade {} to {} unsupported for ARM machines.'.format(
```

Comment:
```suggestion
    title = 'Upgrade from RHEL {} to RHEL {} is unsupported for ARM machines.'.format(
```

Reply:
Not sure whether "unsupported" is correct word - it is just not possible.  @Rezney

Reply:
We might go with something like "impossible", "not possible", "unfeasible"  as well ...

Reply:
not possible it is then!